ace/os_linux.go

```diff
@@ -0,0 +1,63 @@
+// +build linux
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func checkMountImpl(d string, readonly bool) error {
+	mountinfoPath := fmt.Sprintf("/proc/self/mountinfo")
+	mi, err := os.Open(mountinfoPath)
+	if err != nil {
+		return err
+	}
+	defer mi.Close()
+
+	sc := bufio.NewScanner(mi)
+	for sc.Scan() {
+		var (
+			mountID        int
+			parentID       int
+			majorMinor     string
+			root           string
+			mountPoint     string
+			mountOptions   string
+			optionalFields string
```

Comment:
It's probably fine because we only care about (5) but for that same reason we can only parse the first 5 fields (which are fixed) and avoid possible problems. I'll update the PR with that and some unit tests. Thanks!